Recrutify/Recrutify.DataAccess/Repositories/ScheduleRepository.cs

```diff
@@ -29,6 +29,30 @@ public Task<Schedule> GetByDatePeriodAsync(Guid userId, DateTime date, int daysN
             return GetFindFluentByDate(filter, date, daysNum).FirstOrDefaultAsync();
         }
 
+        public Task<IEnumerable<ScheduleSlot>> GetScheduleSlotsByDatePeriodAsync(Guid userId, DateTime dateStartPeriod)
+        {
+            var filter = _filterBuilder.Eq(u => u.UserId, userId);
+            return GetFindFluentScheduleSlotsByDatePeriod(filter, dateStartPeriod).FirstOrDefaultAsync();
+        }
+
+        public Task BulkUpdateScheduleSlotAsync(Guid userId, IEnumerable<DateTime> newDateTime, IEnumerable<DateTime> remoteDateTime)
+        {
+            var updateBuilder = Builders<Schedule>.Update;
+            var filterBuilder = Builders<ScheduleSlot>.Filter;
+
+            var updateModelsWithNewScheduleSlot = newDateTime.Select(ur => new UpdateOneModel<Schedule>(
+                                                    _filterBuilder.Eq(u => u.UserId, userId),
+                                                    updateBuilder
+                                                    .AddToSet(
+                                                        nameof(Schedule.ScheduleSlots),
+                                                        new ScheduleSlot() { AvailableTime = DateTime.SpecifyKind(ur, DateTimeKind.Utc) })));
```

Comment:
If you agree that fe will pass you date (without time) in utc (how it should be) it would't be necessary

Reply:
+